scipy/optimize/_shgo.py

```diff
@@ -925,15 +1055,15 @@ def simplex_minimizers(self):
                 if self.disp:
                     logging.info('=' * 60)
                     logging.info(
-                        'v.x = {} is minimizer'.format(self.HC.V[x].x_a))
-                    logging.info('v.f = {} is minimizer'.format(self.HC.V[x].f))
+                        'v.x = {} is minimiser'.format(self.HC.V[x].x_a))
+                    logging.info('v.f = {} is minimiser'.format(self.HC.V[x].f))
                     logging.info('=' * 30)
 
                 if self.HC.V[x] not in self.minimizer_pool:
                     self.minimizer_pool.append(self.HC.V[x])
 
                 if self.disp:
-                    logging.info('Neighbors:')
+                    logging.info('Neighbours:')
```

Comment:
These are not needed either (while in this case it does not really matter, it increases the diff which makes it harder to review). In general we use American spelling.